scripts/segmentation/test.py

```diff
@@ -88,7 +88,7 @@ def parse_args():
     args = parser.parse_args()
     
     args.ctx = [mx.cpu(0)]
-    args.ctx = [mx.gpu(i) for i in range(args.ngpus)] if args.ngpus > 0 else args.ctx
+    args.ctx = [mx.gpu(i+4) for i in range(args.ngpus)] if args.ngpus > 0 else args.ctx
```

Comment:
what's this `+4` for?

Reply:
While I test the code, my first 4 gpu are used in training another model. So I use `+4` to make the program running in last 4 gpu. I am so sorry that I don't modify it before I push the code.

Reply:
Whenever open a PR, please always double check the diff, especially for such a short one. If there's any mistake, just fix it and push.